src/Navigation/Skill/SkillContent.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import data from "../../resume.json";
+import "./index.css";
+const skillImage = require.context("../../images/");
```

Comment:
Instead of importing `data` and skill images directly inside the component, it would be better to pass them as props from the parent component which is using this component.
Advantage: this component will have less dependency i.e. it will increase this component's reusability and maintainability.

If you want to use this component for a different `data` then it is not possible as this component is strongly-coupled with the data it is importing in this file. On the other hand if data was passed to it via prop then another component can use it by just passing a different data.
A good developer always should create modules with as less tight-couplings or dependencies as possible.